rivery_cli/rivery_session.py

```diff
@@ -366,7 +369,34 @@ def fetch_run_logs(self, **kwargs):
         url = '/activities/runs/logs'
         method = 'get'
         param = {"id": run_id}
-        return self.handle_request(url=url, method=method, params=param, return_full_response=return_full_response)
+
+        query_request = self.handle_request(url=url, method=method, params=param,
```

Comment:
All of this ahould be handles in the command itself and not in the factory. The rivery session should be the basic factory for handling against the API. The command functions are how you add the logic upon the API